pkg/schedule/handler/handler.go

```
@@ -1063,6 +1064,24 @@ func (h *Handler) GetHotBuckets(regionIDs ...uint64) (HotBucketsResponse, error)
 	return ret, nil
 }
 
+// GetRegion returns the region labeler.
+func (h *Handler) GetRegion(id uint64) *core.RegionInfo {
+	c := h.GetCluster()
+	if c == nil {
+		return nil
```

return nil?

---

Do you have any good ideas?

---

Why not return an error